starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,60 @@
+body {
+margin: 0px;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+figure {
+margin: 10px;
+}
+
+.applePies {
+  position: relative;
+  left: 400px;
+  top: -550px;
+  color: white;
+  font-size: 90px;
+
+}
+
+.main {
```

Comment:
This is also making your webpage super wide.  This thing has a default width, and if you move it over to the right like you did, you have to adjust the width and make it skinnier so that it fits.